Assets/Plugins/Source/Editor/ConfigEditors/ConfigEditor.cs

```diff
@@ -247,23 +247,64 @@ public string GetLabelText()
         }
 
         public async Task LoadAsync()
-        {   
+        {
+            // Don't do anything if the config is already loaded.
+            if (config != null)
+                return;
+
             config = await EpicOnlineServices.Config.GetAsync<T>();
```

Comment:
Does this still work with the fact that `Config` caches itself in some circumstances? Needs testing.

Reply:
This _does_ function as expected, because the caching mechanism that happens behind the scenes for `Config` classes is only active in non-editor contexts.